### ISSUE TITLE
Is there a way for SVF to get the vitrual function callsite and all possible call targets on the IR?

### ISSUE THREAD

**Issue by Lqs66** [SVF-tools/SVF#1294]
Dear Developers.
I have been most looking for a way to statically analyze virtual function calls on IR, and I would like to ask if SVF provides such a feature? I would like to call it directly.

**Comment by yuleisui** [comment]
Yes, but based on pointer analysis and CHG you may wish to take a look at "BVDataPTAImpl::onTheFlyCallGraphSolve".

**Comment by Lqs66** [comment]
> Yes, but based on pointer analysis and CHG you may wish to take a look at "BVDataPTAImpl::onTheFlyCallGraphSolve".

Thank you for your reply. It was very helpful to me!
